core/src/test/java/io/grpc/internal/ManagedChannelImplTest.java

```diff
@@ -196,6 +197,7 @@ public void twoCallsAndGracefulShutdown() {
     verify(mockTransport, timeout(1000)).newStream(same(method), same(headers));
     verify(mockStream).start(streamListenerCaptor.capture());
     verify(mockStream).setCompressor(isA(Compressor.class));
+    verify(mockStream).setMessageCompression(anyBoolean());
```

Comment:
Before the transport is created in-line, thus `call` will call `start()` on `mockStream` directly, and it doesn't call `setMessageCompression()` unless compression is set.
After the change, there is a good chance that `call` calls `start()` on a `DelayedStream`, which when starting the real stream (`mockStream`), will always call `setMessageCompression()`.

After thinking about this, this is really not something that needs to be checked in this test. I may just remove `verifyNoMoreInteractions(mockStream);` in the end of this test.